content/desktop/release-notes.md

```diff
@@ -71,6 +71,10 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 - Fixed a bug that caused WSL distro to terminate abruptly. Fixes [for-win/14230](https://github.com/docker/for-win/issues/14230).
 - Fixed an issue that caused WSL to update in each startup. Fixes [for-win/13868](https://github.com/docker/for-win/issues/13868), [for-win/13806](https://github.com/docker/for-win/issues/13806).
 
+### Known Issues
+
+- There is a known issue when authenticating against a registry in the Docker CLI (`docker login [registry address]`) where, if the provided registry address includes a repository/image name (such as `docker login index.docker.io/docker/welcome-to-docker`), the repository part (`docker/welcome-to-docker`) is not normalized and results in credentials being stored incorrectly, which causes subsequent pulls from the registry (`docker pull index.docker.io/docker/welcome-to-docker`) to not be authenticated. To prevent this, don't include any extraneous suffix in the registry address when running `docker login`.
+
```

Comment:
Do we need to call out that this was undocumented behavior, and the expected argument is the name of the registry( hostname (with optional port) `hostname[:port]`), so that users are recommended to use that format, 
or would that be too much info?

cc @dvdksn (in case you have too suggestions for wording)



We should probably also have a note in the engine release notes

Reply:
I considered that, then decided against it since it didn't feel super relevant to this section (listing known issues). I'm happy to change it though, I don't feel super strongly about it.

Adding it to the engine release notes makes sense – I'd also considered a little (temporary) callout in the CLI docs for docker login, or maybe a not-temporary one explaining that the expected input there is not to include anything but the registry hostname/port. Actually, that makes even more since now since we know some users are doing this (Gitlab).

Reply:
Let me add a blurb so we can discuss wording.

Reply:
Did we add release notes for 27.2?

Reply:
I just merged them